certora/harnesses/RewardsManagerHarness.sol

```diff
@@ -1,20 +1,34 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.10;
 
-import "../../contracts/RewardsManager.sol";
+import "../../contracts/RewardsManagerFixed.sol";
```

Comment:
Hi, we're going to examine your specs agains original contract. So could you make several changes, please, and resubmit your merge request?
1. Return original import (`import "../../contracts/RewardsManager.sol";`) so we can examine your specs agains original contract.
2. Rename this harness a little bit and fix scripts according to it. We need it because you added your own methods in harness and if we will merge all changes harnesses together we will have a lot of issue with running a verification.

Reply:
I edited import and deleted some methods. However I left some because I use some rule on them. Is that still an issue? In that case Ill just comment out the corresponding rules and methods